### ISSUE TITLE
Monaco editor references missing file

### ISSUE THREAD

**Issue by Firepup6500** [lafkpages/XL-Replit#67]
**Describe the bug**
When switching to the Monaco editor, the extension tries to load a missing file.

**To Reproduce**

1. Open the extension
2. Enable Monaco
3. Open Console
4. Refresh
5. Note the extension tries to load `chrome-extension://ndifofiicnmolppeagjnekpkplpcjabo/public/vs/editor/editor.main.js`, which doesn't exist/isn't found

**Expected behavior**
The Monaco editor to load

**Screenshots**
![Screenshot (3)](https://github.com/lafkpages/XL-Replit/assets/70233190/ca21e674-05c8-43ca-a30f-7b4d77054a5d)

**Desktop (please complete the following information):**

- OS: `Windows 10 Version 22H2 (Build 19045.3208)`
- Browser: `Chrome 114.0.5735.248 (Official Build) (64-bit) (cohort: Stable)`
- Version: [a40f8f7](https://github.com/lafkpages/XL-Replit/commit/a40f8f7acddc0850199fb0b9707f3c30e46898be)

**Additional context**

**Comment by Firepup6500** [comment]
Ran this:
```bash
git pull && pnpm install && pnpm clean:cache && pnpm build:chrome
```
Removed and re-added the extension to chrome.

I still get:
`GET chrome-extension://ndifofiicnmolppeagjnekpkplpcjabo/public/vs/editor/editor.main.js net::ERR_FILE_NOT_FOUND`

**Comment by Firepup6500** [comment]
Manually trying to follow that file path reveals that `public/vs` is a file in the non-zip, but a proper folder in the zip file. The file the extension calls is available in the zip, but not in the folder.

**Comment by lafkpages** [comment]
Well if the ZIP works I believe you can load that directly on Chrome instead of the folder. But did the build script give any errors?

**Comment by lafkpages** [comment]
Ohhhh wait you said in the non-ZIP it's a file instead of a folder? I might know why. What OS are you on?

**Comment by Firepup6500** [comment]
On Windows 10, though I built the extension with WSL.

**Comment by lafkpages** [comment]
I don't have WSL so I can't test it but do you know if WSL supports symbolic links?

**Comment by Firepup6500** [comment]
After reviewing [this SuperUser post](https://superuser.com/questions/1695779/symlinks-made-in-the-linux-side-of-wsl2-do-not-appear-in-the-windows-side-under), it appears that while WSL does, Windows can't access symlinks that WSL makes.

**Comment by lafkpages** [comment]
Yeah that makes sense. So for now you can try loading the ZIP file, or just unzipping it yourself and loading that.

The only idea I have to fix this would be to simply copy over Monaco instead of symlinking it on WSL, or maybe a command-line option for the build script?

**Comment by Firepup6500** [comment]
Found this package while digging around today, it seems like a good fix for this issue to me:
https://github.com/sindresorhus/is-wsl

**Comment by lafkpages** [comment]
And you're running the build process on WSL, but loading the extension on normal Windows, right?

**Comment by Firepup6500** [comment]
That's correct. I ended up doing that because I got too annoyed with fighting Windows to get PNPM working.

**Comment by lafkpages** [comment]
You probably could've just used NPM although I can't assure you it'd work 100% because you wouldn't be using the lock file.
